scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -439,3 +442,13 @@ <h5 class="modal-title" id="archiveCaseLabel">Confirm archive case</h5>
   </div>
 </div>
 {% endmacro %}
+
+{% macro cnv_report(institute, case) %}
+<div href="#" class="bg-dark list-group-item text-white">
+  <div class="d-flex flex-row flex-fill bd-highlight"></span>
+    <span class="menu-collapsed ml-2">CNV report
+      <a href="{{ url_for('cases.cnv_report', institute_id=institute,
+                        case_name=case.display_name, format='pdf') }}"><i class="fa fa-file-pdf-o"></i></a>
```

Comment:
Nice! Perhaps it would be convenient that the link opens to a new window (target="_blank") ?